docker/etna-base/entrypoints/development.sh

```diff
@@ -19,7 +19,9 @@ if [ -z "$SKIP_BUILD" ]; then
 
   if [ -z "$SKIP_RUBY_SETUP" ]; then
     if [ -z "$SKIP_DB_WAIT" ]; then
-      dockerize -wait tcp://${APP_NAME}_db:5432 -timeout 60s
+      #Sometimes during startup the cpu can be slammed for all the servies, so we need to wait
```

Comment:
This prevents the issue where sometimes on startup the app_fe contains exit with 1 due to timeout (when starting everything at once this can happen)